src/prisma/schema.prisma

```diff
@@ -3166,3 +3167,11 @@ model BuilderCardActivity {
 
   @@unique([builderId])
 }
+
+model ScoutWallet {
+  id        String   @id @default(uuid()) @db.Uuid
```

Comment:
maybe we don't even need an id?

Reply:
yeah, I think it is not necessary. I removed id and added address as an index since we will search by address on login